src/js/utils.js

```diff
@@ -209,3 +209,19 @@ export function when_defined(v, alternativeIn) {
     return is_defined(v) ? v : alternative;
 }
 
+/**
+ * Check if the getStats API for retrieving legacy stats report is supported
+ */
+export function isLegacyStatsReportSupported(pc) {
+    return new Promise(function(resolve) {
+        pc.getStats(function() {
+            resolve(true);
+        }).catch(function(e) {
+            // TypeError if browser does not support legacy stats report
+            if (e instanceof TypeError) {
```

Comment:
Last comment:
Is TypeError the only exception? For the completeness here, you may want to just resolve(false) in case of any exception.

Reply:
Updated to resolve(false) for all exceptions.